driver/kubernetes/factory.go

```diff
@@ -229,6 +232,8 @@ func (f *factory) processDriverOpts(deploymentName string, namespace string, cfg
 			if err != nil {
 				return nil, "", "", false, err
 			}
+		case "provisioningTimeout":
+			deploymentOpt.ProvisioningTimeout, err = strconv.Atoi(v)
```

Comment:
Can we use a `time.Duration` type instead? Would need to do a `time.ParseDuration` here.